src/utils/timestampToEpoch.ts

```diff
@@ -14,9 +14,15 @@ export const timestampToEpoch = (timestamp: BigNumberish) => {
 };
 
 export const getEpochsBetweenTimestamps = (tFrom: BigNumberish, tTo: BigNumberish) => {
-  const epochFrom = timestampToEpoch(tFrom);
-  if (!epochFrom) return;
+  let epochFrom = timestampToEpoch(tFrom);
   const epochTo = timestampToEpoch(tTo);
+  if (!epochTo) return [];
```

Comment:
This restart the conversation about error handling... Once again I think it's better to throw errors in case of unexpected behavior rather than handling errors with default values.
As a user, I prefer to get an error or a message telling me that something went wrong instead of having the default value without any further information.

Reply:
This is not an unexpected behavior, if you use 2 timestamps (morphoCompoundDeployment, morphoCompoundDeployment + 1000, there is no epochs

Reply:
That's correct, and I think it's the reason why @oumar-fall and I feel like we should throw an error instead of silently continue